### PR TITLE
Allow new Hydrogen options from #1940 to be customized

### PR DESCRIPTION
**Pull request by aaronraimist**

### CODE REVIEW

**Comment by JokerGermany** [comment]
Why is there a hardcoded `gatewayUrl": "https://matrix.org",` in it? is that necessary?

File: roles/matrix-client-hydrogen/defaults/main.yml [hunk 1]
```diff
@@ -31,7 +31,14 @@ matrix_client_hydrogen_systemd_required_services_list: ['docker.service']
 matrix_client_hydrogen_self_check_validate_certificates: true
 
 # config.json
+matrix_client_hydrogen_push: {
+  "appId": "io.element.hydrogen.web",
+  "gatewayUrl": "https://matrix.org",
+  "applicationServerKey": "BC-gpSdVHEXhvHSHS0AzzWrQoukv2BE7KzpoPO_FfPacqOo3l1pdqz7rSgmB04pZCWaHPz7XRe6fjLaC-WPDopM"
+}
```
Comment (spantaleev) [inline roles/matrix-client-hydrogen/defaults/main.yml:38]:
```suggestion
matrix_client_hydrogen_push:
  appId: io.element.hydrogen.web
  gatewayUrl: https://matrix.org
  applicationServerKey: "BC-gpSdVHEXhvHSHS0AzzWrQoukv2BE7KzpoPO_FfPacqOo3l1pdqz7rSgmB04pZCWaHPz7XRe6fjLaC-WPDopM"
```

Isn't it better to keep this more YAML-like?

**Comment by spantaleev** [comment]
Thanks! :+1:

**Comment by aaronraimist** [comment]
@JokerGermany I'm not sure what you mean. It's not hardcoded. I created this PR specifically so that you can change that and the other options. If you mean why did https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1940 hardcode it I don't know, I assume they wanted to use it.

**Pull request merged**